_posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreaders.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Learning common keyboard shortcuts for screen readers"
+description: "Keyboard shortcuts are hard to learn. Here are some other ways that can help that learning."
+author: anevins12
+published: true
+categories:
+  - Assistive Technology
+further_reading:
+  - url: https://dequeuniversity.com/screenreaders/
+    title: "Screen Reader Keyboard Shortcuts and Gestures"
+    source: Deque University
+  - url: https://gist.github.com/anevins12/a9cc6548fcbab5f55c1dc3c7bbb0f4bd
+    title: "Source JSON file of the mindmap that can be imported into http://app.mindmapmaker.org/"
+    source: Github
+---
+
+Keyboard shortcuts are often presented to us in a list or a table. Lists and tables are great for some people, while others may be overwhelmed from the sheer number of commands to learn. Sometimes a more visual approach could help us learn.
+
+## Learning visually helps
+Creating a mind map of keyboard shortcuts can help learn those shortcuts easier than lists and tables. 
+
+Here's a mind map of the common keyboard shortcuts in popular screen readers: ![Mind map of common keyboard shortcuts for screen readers](/img/posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreader/common-keyboard-shortcuts-mindmap.png)
+
+That mind map shows some shortcuts across screen readers by tasks but is not exhaustive. An example of one task is to 'list all landmarks'. This task then branches off into JAWS, NVDA, VoiceOver iOS, TalkBack and Narrator screen readers. Each then branches into their own shortcut:
+- List all landmarks -> JAWS -> R or ;
+- List all landmarks -> NVDA -> D
+- List all landmarks -> Narrator -> D
+- List all landmarks -> VoiceOver iOS -> Two finger twist (the rotor menu)
+- List all landmarks -> TalkBack -> Swipe right (the local context menu)
+
+The shortcuts are taken from [Deque on shortcuts and gestures](https://dequeuniversity.com/screenreaders/).
+
+## Found a cool shortcut? Look it up
+Looking up videos on how to do those shortcuts can reinforce that learning. Trying out yourself on a device is even better. Android devices have TalkBack, iOS and MacOS have VoiceOver and Microsoft has Narrator.
```

Comment:
- provide a link to a video, rather than making people have to search themselves
- macOS does not have a capital M.